js_modules/dagit/packages/core/src/app/FallthroughRoot.tsx

```diff
@@ -49,10 +50,12 @@ const FinalRedirectOrLoadingRoot = () => {
   }
 
   // If we have exactly one job, route to the job's overview / graph tab
-  const reposWithJob = allRepos.filter((r) => r.repository.pipelines.length > 0);
+  const reposWithJob = allRepos.filter((r) =>
+    r.repository.pipelines.some((p) => p.name !== __ASSET_GROUP),
+  );
   if (reposWithJob.length === 1) {
     const repo = reposWithJob[0];
-    const job = repo.repository.pipelines[0];
+    const job = repo.repository.pipelines.filter((p) => p.name !== __ASSET_GROUP)[0];
```

Comment:
We sure its never possible for a repository not named `__ASSET_GROUP`  to have 0 pipelines? That's what this was preventing before, and now we'd be setting up job = undefined.

Reply:
Ahh interesting you're right... I think this check needs to be changed slightly 👍